Dockerfile.template

```diff
@@ -22,6 +22,8 @@ RUN set -ex; \
 # wget: error getting response: Connection reset by peer
 	apk add --no-cache --virtual .fetch-deps \
 		curl \
+# https://github.com/moby/moby/pull/35697
+		pigz \
 		tar \
 	; \
```

Comment:
I think we should move this to the end of the list, and make sure the comment includes `pigz` as well so that the following patch can be applied to `update.sh` as well (which when run, should make this a template-only change PR):

```dockerfile
	apk add --no-cache --virtual .fetch-deps \
		curl \
		tar \
# pigz: https://github.com/moby/moby/pull/35697
		pigz \
	; \
```

```diff
diff --git a/update.sh b/update.sh
index 496dcf4..36a5e48 100755
--- a/update.sh
+++ b/update.sh
@@ -72,6 +72,11 @@ for version in "${versions[@]}"; do
 
 	alpine="${alpineVersion[$version]:-$defaultAlpineVersion}"
 
+	majorVersion="${fullVersion%%.*}"
+	minorVersion="${fullVersion#$majorVersion.}"
+	minorVersion="${minorVersion%%.*}"
+	minorVersion="${minorVersion#0}"
+
 	for variant in \
 		'' git dind \
 		windows/windowsservercore-{1709,ltsc2016} \
@@ -92,6 +97,11 @@ for version in "${versions[@]}"; do
 			-e 's!%%ARCH-CASE%%!'"$(sed_escape_rhs "$archCase")"'!g' \
 			"$template" > "$df"
 
+		# pigz (https://github.com/moby/moby/pull/35697) is only 18.02+
+		if [ "$majorVersion" -lt 18 ] || { [ "$majorVersion" -eq 18 ] && [ "$minorVersion" -lt 2 ]; }; then
+			sed -ri '/pigz/d' "$df"
+		fi
+
 		if [[ "$variant" == windows/* ]]; then
 			winVariant="$(basename "$variant")"
 
```

(also happy to commit these changes directly to the PR branch if you'd rather I take over from here)